packages/react/tabs/src/Tabs.stories.tsx

```diff
@@ -308,21 +405,32 @@ const triggerClass = css({
 
 const RECOMMENDED_CSS__TABS__CONTENT = {
   flexGrow: 1,
-};
-
-const contentClass = css({
-  ...RECOMMENDED_CSS__TABS__CONTENT,
-
   padding: '1em',
```

Comment:
rather than moving all this to the "recommended" css, I think instead what you wanted to do was to extend contentClass in animatedContentClass.

Reply:
Does my comment make sense here? I'm not sure I made it very clear 😀  Feel free to ping me if that's the case.